third_party/gpus/find_cuda_config.py

```diff
@@ -318,10 +318,6 @@ def get_header_version(path):
     # cuBLAS uses the major version only.
     cublas_version = header_version.split(".")[0]
 
-    if not _matches_version(cuda_version, cublas_version):
```

Comment:
Is it supported to use different major version of CUDA and cuBLAS?

Reply:
CUDA 10.2 comes with mismatched versions between CUDA and cuBLAS.

Reply:
That's not what I see for the version that we use internally. The situation might have changed.
Could you provide the exact version of the CUDA SDK that you are using? Thanks.

Reply:
@alecyu2018 Can you please check @chsigg's comments and keep us posted. Thanks!